### ISSUE TITLE
Update the cookie dependency

### ISSUE THREAD

**Issue by bbigras** [iron/iron-sessionstorage#10]
I can't wait to have at least 0.7.3 to use the new "same domain" attribute. See
https://github.com/alexcrichton/cookie-rs/issues/80

**Comment by untitaker** [comment]
Right, this appears to be a bit more involved though. If you have the time
please open a PR.

**Comment by bbigras** [comment]
OK. I did some progress on the redis part. I'll take a look at it.

**Comment by beatgammit** [comment]
Sort of related, but it would be nice to update all of the dependencies. I would open another issue, but there's nothing in particular that I really want from the other dependencies right now besides the cookie dependency.
